docs/changelog.md

```diff
@@ -1,5 +1,42 @@
 # Changelog
 
+## 1.12.0 (2025-05-13)
+
+
+### Major changes
+
+- The build system has been updated to use `pyproject.toml` instead of `setup.py`
+- Support for Python 3.9 has been dropped. The minimum supported Python version is now
+  3.10; the minimum supported NumPy version is 1.22; the minimum supported `ndonnx`
+  version is 0.10.1.
```

Comment:
```suggestion
  3.10; the minimum supported NumPy version is 1.22.
```
I find this special-casing for ndonnx a bit bizarre. We're not testing against anything other than the latest versions.
As a general rule, I would not advertise what I do not test.